README.md

```diff
@@ -8,17 +8,29 @@
 
 Windows, OS X & Linux::
 
-- Run docker container & install dependencies
+## Run project
+
+- Run docker container
 ```sh
 docker-compose up -d
 docker-compose exec -u web web bash
-./bin/console doctrine:migra:migra
+```
+
+- Install dependencies
+```bash
+composer install -o
+yarn install
```

Comment:
This command is already in `Build assets` section

```suggestion
```

Reply:
Thank you very much. The README has been modified to clarify this. Now it is detailed how to install the dependencies and how to do the build separately.

Reply:
Thank you very much. The README has been modified to clarify this. Now it is detailed how to install the dependencies and how to do the build separately.